internal/cmd/scan/scan.go

```diff
@@ -128,7 +128,7 @@ $ debricked scan . `+exampleFlags)
 	cmd.Flags().BoolVarP(&passOnDowntime, PassOnTimeOut, "p", false, "pass scan if there is a service access timeout")
 	cmd.Flags().BoolVar(&noResolve, NoResolveFlag, false, `disables resolution of manifest files that lack lock files. Resolving manifest files enables more accurate dependency scanning since the whole dependency tree will be analysed.
 For example, if there is a "go.mod" in the target path, its dependencies are going to get resolved onto a lock file, and latter scanned.`)
-	cmd.Flags().BoolVar(&noFingerprint, FingerprintFlag, false, "enables fingerprinting for undeclared component identification. Can be run as a standalone command [files fingerprint] with more granular options. Will be default in an upcoming major release.")
+	cmd.Flags().BoolVar(&noFingerprint, FingerprintFlag, false, "disables fingerprinting for undeclared component identification. Can be run as a standalone command [files fingerprint] with more granular options.")
```

Comment:
Isn't it a bit backwards to state "Can be run as a standalone command [files fingerprint] with more granular options." when this flag now **disables** fingerprint? It would make more sense to share this information when we inform the user that they cannot fingerprint in scans if they are not enterprise.